build.gradle

```diff
@@ -257,13 +258,7 @@ subprojects {
 
 
   ext {
-    applyAliasingChecker = { ->
-      checkerFramework {
-        checkers += 'org.checkerframework.common.aliasing.AliasingChecker'
-      }
-    }
-
-    applyIntellijPlugin = { Map params ->
+    addIntellijToCompileClasspath = { Map params ->
```

Comment:
In fact, see https://github.com/JetBrains/gradle-intellij-plugin/issues/494 - I'd rather want it to be "adding a dependency" than "applying a plugin" for all modules other than root project & `uiTests` (which actually build their own plugins)